contracts/token/ERC20/TokenTimelock.sol

```diff
@@ -9,10 +9,10 @@ import "./SafeERC20.sol";
  * beneficiary to extract the tokens after a given release time
  */
 contract TokenTimelock {
-  using SafeERC20 for ERC20Basic;
+  using SafeERC20 for StandardToken;
 
   // ERC20 basic token contract being held
-  ERC20Basic public token;
+  StandardToken public token;
```

Comment:
Shouldn't this (and all other references to the type in this contract) be an `ERC20`?